app/actions/local/scheduled_post.test.ts

```diff
@@ -118,3 +118,37 @@ describe('handleScheduledPosts', () => {
         expect(models).toBeUndefined();
     });
 });
+
+describe('handleUpdateScheduledPostErrorCode', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should update scheduled post error code successfully', async () => {
```

Comment:
In this test we are not verifying the error has been updated. You should verify the model returned from `handleUpdatescheduledPostErrorCode` has the new error.

Another thing that would be nice to test (not sure if possible due to the caching of WatermelonDB) is the following:
- If prepareRecordsOnly is set, getting the value from the database should show the error unchanged
- If prepareRecordsOnly is not set, getting the value form the database should show the error changed

Reply:
Created a ticket to take this up later: https://mattermost.atlassian.net/browse/MM-63461